OpenRA.Mods.Common/Scripting/Global/MediaGlobal.cs

```diff
@@ -207,6 +207,13 @@ public void Debug(string text)
 			Game.Debug(text);
 		}
 
+		[Desc("Write a message into debug.log")]
+		public void DebugLog(string text)
```

Comment:
No, that uses `Game.Debug` which displays it in-game. If you want to dump a large amount of data the chat log would get spammed and you can't search it.